src/main/java/services/WriterService.java

```diff
@@ -0,0 +1,8 @@
+package services;
+
+import java.util.List;
+import model.Product;
+
+public interface WriterService {
+    void writeToFile(String toFile, List<Product> products);
```

Comment:
```suggestion
    void writeToFile(String toFile, String report);
```
you should pass here already formed report